app/experimenter/docs/openapi-schema.json

```diff
@@ -7613,39 +7613,49 @@
               "application/json": {
                 "schema": {
                   "properties": {
-                    "owner": {
-                      "type": "string",
-                      "readOnly": true
+                    "audience": {
+                      "enum": [
+                        "AUDIENCE 1",
+                        "AUDIENCE 2"
```

Comment:
Guess I haven't thought of this... since this is a JSON file, how will we be able to grab these enum values from the nimbus-shared library? Unless we have some other .js file which generates the openapi-schema.json files and adds a bit of scripting flair.

Reply:
The `openapi-schema.json` is auto generated from the python code that implements the APIs, it's not something we manipulate by hand.  The dummy constants appear here right now:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants.py#L68-L70

and these are what will be replaced by the values in `nimbus-shared`.  The API JSON will update itself automatically to reflect it.